boards/arduino-leonardo/src/lib.rs

```diff
@@ -172,20 +171,23 @@ pub mod pwm {
     pub use atmega32u4_hal::pwm::*;
 }
 
+pub type Baudrate = hal::usart::Baudrate<hal::clock::MHz16>;
```

Comment:
I'd prefer just re-exposing the usart module from the HAL:

```rust
pub use hal::usart;
```

and then importing it in the examples:

```rust
pub use arduino_leonardo::usart;

// ...
usart::Baudrate::new(...);
```

This is what we're doing for all other modules so I would argue it's best to use the same pattern for serial as well.

Reply:
done, should I fix for other imports?

Reply:
Not sure what other imports you're talking about?

Reply:
Looks like I made same as `arduino_leonardo::Pins` but is defined under leaonardo not in generic. I review other and they are correct.

Reply:
Ah, `Pins` is the exception :D  The reason is that `Pins` is so fundamental that it's just annoying when hidden behind a module.  The type-alias for e.g. `Serial` is similar though I am less sure if we want to keep that in the long run.